Formula/swi-prolog.rb

```diff
@@ -0,0 +1,93 @@
+class SwiProlog < Formula
+  homepage "http://www.swi-prolog.org/"
```

Comment:
Please add a `desc "Description of this formula"` line before the `homepage` line. See the [Formula Cookbook](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Formula-Cookbook.md#fill-in-the-homepage) and compare with other formulae that have a description (all should have one in this repository).